packages/gestalt-core/build.js

```diff
@@ -57,7 +57,14 @@ const cssModules = (options = {}) => {
         scopeNames[hash] = classnameBuilder.getMinifiedClassname(hash);
       }
 
-      return scopeNames[hash];
+      // if the DEVMODE environment variable is set, then don't minify the classnames
+      const isDevMode = process.env.DEVMODE === 'true';
+      let minifiedName = scopeNames[hash];
```

Comment:
In general, if a variable only changes on a single conditional, it's best to use a ternary so we can use `const`. That prevents the variable from being reassigned later.

Reply:
Ah, I did this tweak, and changing the NODE_ENV variable causes things to fail. Not sure why. It may be being somewhere internally. So it seems a tad more complex to debug. See error below:

I kept the DEVMODE env variable, but changed to a ternary for simplicity

<img width="854" alt="image" src="https://github.com/pinterest/gestalt/assets/5509813/7f458755-cc50-4712-a716-ca75753a1c56">